packages/react-core/src/components/Page/examples/PageVerticalNavUsingPageHeaderComponent.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import {
   Page,
-  PageHeader,
-  PageHeaderTools,
   PageSidebar,
   PageSection,
   PageSectionVariants
 } from '@patternfly/react-core';
+import {
+  PageHeader,
+  PageHeaderTools
+} from '@patternfly/react-core/deprecated';
```

Comment:
Just to match the import in the MD file/the imports in the other files:

```suggestion
} from '@patternfly/react-core/dist/esm/deprecated';
```